website/source/intro/getting-started/jobs.html.md

```diff
@@ -143,8 +174,9 @@ or to change the count of a task group to scale with load.
 For now, edit the `example.nomad` file to uncomment the count and set it to 3:
 
 ```
-# Control the number of instances of this group.
-# Defaults to 1
+# The "count" parameter specifies the number of the task groups that should
+# be running under this group. This value must be non-negative and defaults
+# to 1.
```

Comment:
Same here -- I think we can keep this succinct rather than a full representation of what's actually in example.nomad, but going to defer to @dadgar on this.